### PR TITLE
Re-do debian package

### PR DESCRIPTION
**Pull request by romcheg**
- Ubuntu 18.04 support
- Use debconf for necessary configuration
- Build in a docker container to allow building anywhere
- Settings stored in /etc/ralph
- Variable data stored in /var
- system.d service

Co-authored: Roman Prykhodchenko (@romcheg)
Co-authored: Szymon Jasinski (@szymi-)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,13 +4,24 @@ TEST?=ralph
 
 package: build-package upload-package
 
-build-package:
+build-package-docker:
```
Comment (szymi-) [inline Makefile]: Is this still needed?

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,10 @@
+ralph (3.0.0-30) UNRELEASED; urgency=medium
```
Comment (szymi-) [inline debian/changelog]: This is not needed

Comment (romcheg) [reply debian/changelog]: Nope. That's removed now.

**Comment by romcheg** [comment]
GPG key must be configured for git 
`git config --global user.signingkey KEYID`

File: docker_ng/Dockerfile-deb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:bionic
+
+MAINTAINER PyLabs pylabs@allegro.pl
```
Comment (MarekBleschke) [inline docker_ng/Dockerfile-deb:3]: We leave here pylabs or should it be `opensource@allegro.pl`?

File: Makefile [hunk 4]
```diff
@@ -2,14 +2,24 @@ TEST?=ralph
 
 .PHONY: test flake clean coverage docs coveralls
 
-package: build-package upload-package
+release-new-version: new_version = $(shell ./get_version.sh generate)
+release-new-version:
```
Comment (MarekBleschke) [inline Makefile:9]: I think we could add some small comments to those 3 new commands, especially when to use build-package vs build-snapshot-package. @romcheg what do you think?

File: docker_ng/Dockerfile-deb (hunk 3 above)
Comment (vi4m) [reply docker_ng/Dockerfile-deb:3]: Leave pylabs for now.

**Review by Blejwi — APPROVED** [review]

**Review by MarekBleschke — APPROVED** [review]

**Pull request merged**